test/e2e/storage/vsphere/vsphere_volume_fstype.go

```diff
@@ -29,6 +29,7 @@ import (
 	vsphere "k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere"
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/storage/utils"
+	"k8s.io/kubernetes/test/e2e/storage/vsphere/bootstrap"
```

Comment:
Doesn't this unused import give make error?

Reply:
Thanks for catching. Editor didn't give any error, so I failed to notice. Fixed now. Ran make successfully.